targets/generic

```diff
@@ -44,6 +44,8 @@ config 'CONFIG_PACKAGE_ATH_DEBUG=y'
 config '# CONFIG_KERNEL_IP_MROUTE is not set'
 config '# CONFIG_KERNEL_IPV6_MROUTE is not set'
 
+config 'CONFIG_COLLECT_KERNEL_DEBUG=y'
```

Comment:
Does this have an effect on the build time?

Reply:
The kernel is compiled in `openwrt/build_dir/target-<target>/linux-<arch>/linux-<version>/vmlinux` anyway. This file gets zipped and is placed into the output directory.
Impact on the build time should be below 1s.

Reply:
For large kernels like x86, the bzip2 takes ~18s on my machine. An increase of 10% for the incremental build time for a single device :/

Reply:
I don't have a good idea how to proceed.
- Some kind of escape hatch for fast incremental builds during development? Might lead to lengthy rebuilds on CONFIG_COLLECT_KERNEL_DEBUG changes though, not sure
- Changing the compression method (-> OpenWrt patch; maybe make this configurable for upstream)

Reply:
Okay, I just tested a few compressions tools (standard Debian Buster packages) with a x86-64 build:
- kernel-debug.tar: 376M
- kernel-debug.tar.gz: 101M, compressed in ~12s
- kernel-debug.tar.bz2: 91M, compressed in ~15s
- kernel-debug.tar.xz: 57M, compressed in ~101s
- kernel-debug.tar.zst: 86M, compressed in ~1s

I think we have a clear winner - I'll look into bringing zstd support into OpenWrt soon.

Reply:
https://patchwork.ozlabs.org/project/openwrt/list/?series=176744